.github/dependabot.yml

```diff
@@ -0,0 +1,26 @@
+version: 2
+updates:
+
+    -   package-ecosystem: "gradle"
+        directory: "/"
+        schedule:
+            interval: "daily"
+        reviewers:
+            - "quanticc"
+            - "austinv11"
+            - "darichey"
```

Comment:
Add @NovaFox161  ? Idk if they want

Reply:
Okay, removed darichey as requested. @NovaFox161 should i add you to the reviewers?

Reply:
No i mean add novafox, not remove darichey

Reply:
ohh okay sorry, i gonna revert the last commit.